deploy/examples/create-test-cluster.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+#set -eEx
+KUBECTL="minikube kubectl --"
+ROOK_EXAMPLES_DIR="."
+
+wait_for_ceph_cluster() {
+    echo "Waiting for ceph cluster"
+    WAIT_CEPH_CLUSTER_RUNNING=20
+    until [[ $(kubectl -n rook-ceph exec -it deploy/rook-ceph-tools -- ceph status) =~ .*'health: HEALTH_OK'.* ]]; do
```

Comment:
kubect wait command

Reply:
in this case we are waiting a specific field value on the cluster... not sure if  `kubectl wait` can be used